services/oc-guytonmodel/VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
once the service is ready to be pushed you will need to update to version 1.0.0 using ```make version-service-major```